userbenchmark/cpu/run_config.py

```diff
@@ -59,7 +60,13 @@ def run_config(config: TorchBenchModelConfig, metrics: List[str], dryrun: bool=F
                 # load the model instance within current process
                 model = load_model(config)
             # get the model test metrics
-            result: TorchBenchModelMetrics = get_model_test_metrics(model, metrics=metrics)
+            if profile:
+                print("[INFO] Collecting Profiling logs...")
+                with profiler.profile(activities=[profiler.ProfilerActivity.CPU]) as prof, profiler.record_function("model_inference"):
+                    result: TorchBenchModelMetrics = get_model_test_metrics(model, metrics=metrics)
+                print(prof.key_averages().table(sort_by="self_cpu_time_total", row_limit=-1))
```

Comment:
Can we also support to write the profiling results to a json file?

Reply:
> Can we also support to write the profiling results to a json file?

It's not easy to transfer this table to Json file. As Json file is not good at showing table-content in format.
The `torch.profile` provide a function for chrome_trace json file, which can only open in Chrome Browser or VScode. 
And `prof.key_averages().table()` is suggested print out directly, as the line breaker `\n` will be printed at the end each line.
It looks like the command line output is the most suggested way.
```
>>> prof.key_averages().table()
'-------------------  ------------  ------------  ------------  ------------  ------------  ------------  \n               Name    Self CPU %      Self CPU   CPU total %     CPU total  CPU time avg    # of Calls  \n-------------------  ------------  ------------  ------------  ------------  ------------  ------------  \n    model_inference       100.00%     124.000us       100.00%     124.000us     124.000us             1  \n-------------------  ------------  ------------  ------------  ------------  ------------  ------------  \nSelf CPU time total: 124.000us\n'
>>> prof.key_averages()

```

Reply:
ok, seems it profile for all iterations, is it our expected behavior? refer profile in `run.py`, https://github.com/pytorch/benchmark/blob/main/run.py#L187